english/fus-roh-dah/fus-roh-dah.md

```diff
@@ -0,0 +1,7 @@
+Wait a second. you're the dragonborn! These things are specs of dust compared to you. In a second you can send them all to oblivion.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.